portfolio/src/main/webapp/bandGen.js

```diff
@@ -0,0 +1,22 @@
+function getMood() {
+    let menu = document.getElementById("mood");
+    let mood = menu.options[menu.selectedIndex].text;
+    return mood;
+}
+
+function generateBand() {
+    let mood = getMood();
+    let band = addBands();
+    let bandContainer = document.getElementById("bandContainer");
+    let rand = Math.floor(Math.random() * Math.floor(2));
+    bandContainer.innerHTML = band[mood][rand];
+}
+
+function addBands(){
+    let band = [];
```

Comment:
I guess you're trying to instantiate an object here, not an array. So probably `let band = {}` instead